target/linux/ath79/image/common-mikrotik.mk

```diff
@@ -1,8 +1,21 @@
 define Device/mikrotik
 	DEVICE_VENDOR := MikroTik
+	DEVICE_PACKAGES := rbextract rbcfg
 	BOARD_NAME := routerboard
 	LOADER_TYPE := elf
 	KERNEL := kernel-bin | append-dtb | lzma | loader-kernel
 	KERNEL_INITRAMFS := kernel-bin | append-dtb | lzma | loader-kernel
 	FEATURES += minor
 endef
+
+define Device/rb-nor-flash-16M
```

Comment:
Please keep lower case here: rb-nor-flash-16m

Reply:
Assuming "rb" means "routerboard", please use "routerboard" in the name.
As this is specific to mikrotik, I'd even use "mikrotik_routerboard_nor_16m".

Apart from the naming, I do not think the common node is necessary here at this point.
At the moment, we have just one device, so I'd add the content of this node and the one below to the device.

I'd prefer to create common nodes later when they are needed, so they can be shaped directly onto this need.

Reply:
Agree, let's do it when needed.

Reply:
But let's keep the "common-mikrotik.mk" file; I've got already a couple of MikroTik devices I want to add to ath79 as soon this PR gets accepted :)

Reply:
Yes, keep that and also the "Device/mikrotik" definition.

I'm actually a fan of common definitions and will have them created later when we can better judge on their content.